packages/storybook-addon-designs/src/manager/components/Figma.tsx

```diff
@@ -7,7 +7,7 @@ import { IFrame } from "./IFrame";
 import { FigmaConfig, IFrameConfigBase } from "../../config";
 
 export const figmaURLPattern =
-  /https:\/\/([w.-]+.)?figma.com\/(file|proto)\/([0-9a-zA-Z]{22,128})(?:\/.*)?$/;
+  /https:\/\/[\w\.-]+\.?figma.com\/([\w-]+)\/([0-9a-zA-Z]{22,128})(?:\/.*)?$/;
```

Comment:
As Codacy annotates, the first `\.` is totally unnecessary. However, this is from Figma docs so I kept that as-is.

Reply:
I ended up following Codacy's advice anyway. Easier to just keep it happy than to work around it.